patches/@reduxjs+toolkit+1.5.0.patch

```diff
@@ -0,0 +1,74 @@
+diff --git a/node_modules/@reduxjs/toolkit/dist/redux-toolkit.cjs.development.js b/node_modules/@reduxjs/toolkit/dist/redux-toolkit.cjs.development.js
+index 96ead94..c5492d3 100644
+--- a/node_modules/@reduxjs/toolkit/dist/redux-toolkit.cjs.development.js
++++ b/node_modules/@reduxjs/toolkit/dist/redux-toolkit.cjs.development.js
+@@ -204,7 +204,7 @@ function (_Array) {
+ 
+   var _proto = MiddlewareArray.prototype;
+ 
+-  _proto.concat = function concat() {
++  Object.defineProperty(_proto, 'concat', { value: function concat() {
```

Comment:
Huh. I don't quite follow what's going on here. Is this an array class that is extending `Array` and redefining `concat`?

Could we propose this upstream? They seem to be fairly responsive maintainers.

Reply:
your understanding is correct. its fine for them to do that (none of my business), but the override mistake doesnt allow them to do it via assignment. thus this.
yes, we usually PR these patches upstream. but we dont let that block us so we write patches as well.

Reply:
Got it. Using a patch seems fine for now, but having that work started by creating a PR would be great. Then we could delete the patch on the next update, rather than re-write it each time. Especially for packages like this that are updated relatively frequently.